operator/json.go

```diff
@@ -12,8 +12,8 @@ type JSONOperator struct {
 }
 
 type JSONConfiguration struct {
-	Fields          []string `json:"fields"`
-	TargetFieldName string   `json:"targetFieldName"`
+	Fields          map[string]bool `json:"fields"`
```

Comment:
@bahadirbb I think it could still be []string, but you make a map outside the loop instead. It feels a bit strange to have it as a map[string]bool, unless the bool actually means something, in which case maybe it should be a struct with FieldConf or something to make it clear what it means.

Reply:
I see your point, so I am going to make it []string and use it as a map inside the code